File: app/src/main/java/org/rivchain/cuplink/DefaultSdpObserver.java
package org.rivchain.cuplink;

import org.webrtc.SdpObserver;
import org.webrtc.SessionDescription;


class DefaultSdpObserver implements SdpObserver {
    @Override
    public void onCreateSuccess(SessionDescription sessionDescription) {
        // nothing to do
    }

    @Override
    public void onSetSuccess() {
        // nothing to do
    }

    @Override
    public void onCreateFailure(String s) {
        // nothing to do
    }

    @Override
    public void onSetFailure(String s) {
        // nothing to do
    }
}

File: app/src/main/java/d/d/meshenger/call/DefaultSdpObserver.kt
package d.d.meshenger

import org.webrtc.SdpObserver
import org.webrtc.SessionDescription

internal open class DefaultSdpObserver : SdpObserver {
    override fun onCreateSuccess(sessionDescription: SessionDescription) {
        // nothing to do
    }

    override fun onSetSuccess() {
        // nothing to do
    }

    override fun onCreateFailure(s: String) {
        // nothing to do
    }

    override fun onSetFailure(s: String) {
        // nothing to do
    }
}